base/abstractarray.jl

```diff
@@ -325,15 +325,15 @@ end
 checkbounds(A::AbstractArray) = checkbounds(A, 1) # 0-d case
 
 """
-    checkbounds_indices(Bool, IA, I)
+    checkbounds_indices(Bool, A, IA, I)
```

Comment:
Eh, I think I'd have a hard time scolding @timholy for using a function he wrote. We can see what he says. I can look to see how DistributedArrays uses it.